### PR TITLE
Make LoTAS components translatable

### PR DESCRIPTION
**Pull request by ScribbleTAS**
- [Fabric] Fixed crash when using F3+Esc
- [Fabric]  Added translation loading via LoTASLanguageManager
- Added 1.20
# TODO
## Fabric
- [x] LoTAS Gui
- [x] Keybinds
- [x] AiManip
- [x] Config
- [x] DragonManip
- [x] InfoHud
- [x] Loadstate
- [x] Spawnmanip
- [x] DropManip
   - [x] Deadbush
   - [x] Glowstone
   - [x] Gravel
   - [x] Leaf
   - [x] Ore
   - [x] Plants
   - [x] SeaLantern
   - [x] Drowned
   - [x] Fish
   - [x] Monster
   - [x] Nether
   - [x] Passive
   - [x] Zombie
   - [x] Bartering
## Forge
- [x] LoTAS Gui
- [x] Keybinds
- [x] AiManip
- [x] Config
- [x] DragonManip
- [x] InfoHud
- [x] Loadstate
- [x] Spawnmanip
- [x] DropManip
   - [x] Deadbush
   - [x] Glowstone
   - [x] Gravel
   - [x] Leaf
   - [x] Ore
   - [x] Plants
   - [x] SeaLantern
   - [x] Monster
   - [x] Nether
   - [x] Passive
   - [x] Zombie

### CODE REVIEW

**Comment by George417** [comment]
nice

File: LoTAS-Fabric/src/main/java/de/pfannekuchen/lotas/core/utils/KeybindsUtils.java [hunk 1]
```diff
@@ -108,12 +108,11 @@ public static void tickKeyEvent() {
 		}
 		// Autostrafe auto rotation handling
 		if (wasPressed != holdStrafeKeybind.isDown() && wasPressed == true) {
-			KeyMapping.set(Minecraft.getInstance().options.keyRight.getDefaultKey(), false);
+			Minecraft mc = Minecraft.getInstance();
```
Comment (ScribbleTAS) [inline LoTAS-Fabric/src/main/java/de/pfannekuchen/lotas/core/utils/KeybindsUtils.java:112]: Fixed some resource leak warnings

File: LoTAS-Fabric/build.gradle [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import groovy.json.JsonOutput
 
-version = "2.1.0"
+version = "2.1.2"
```
Comment (PancakeTAS) [inline LoTAS-Fabric/build.gradle]: wat

File: LoTAS-Fabric/src/main/java/de/pfannekuchen/lotas/core/utils/LoTASLanguageManager.java
Comment (PancakeTAS) [inline LoTAS-Fabric/src/main/java/de/pfannekuchen/lotas/core/utils/LoTASLanguageManager.java:1]: you nailed implementing the i18n _standard_

Comment (ScribbleTAS) [reply LoTAS-Fabric/src/main/java/de/pfannekuchen/lotas/core/utils/LoTASLanguageManager.java:1]: I don't know if this is sarcastic or a compliment

**Review by PancakeTAS — APPROVED** [review]
awesome, i will never read through this code again

**Comment by ScribbleTAS** [comment]
> awesome, i will never read through this code again

Never say never

**Review by PancakeTAS — APPROVED** [review]

**Review by PancakeTAS — APPROVED** [review]

**Review by PancakeTAS — APPROVED** [review]

**Pull request merged**